packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -275,10 +275,14 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget {
         this.toHide.push(this.on(WebviewMessageChannels.doUpdateState, (state: any) => {
             this._state = state;
         }));
-        this.toHide.push(this.on(WebviewMessageChannels.didFocus, () =>
+        this.toHide.push(this.on(WebviewMessageChannels.didFocus, () => {
             // emulate the webview focus without actually changing focus
-            this.node.dispatchEvent(new FocusEvent('focus'))
-        ));
+            this.node.dispatchEvent(new FocusEvent('focus'));
+            // We have to dispatch 'mousedown' event in the <webview> in order
+            // to inform 'phosphor' to close the menu if open
+            // ('phosphor' closes the menu when a 'mousedown' event emitted)
+            this.node.dispatchEvent(new MouseEvent('mousedown'));
```

Comment:
It looks weird to dispatch `mousedown` event when webview is focused.
I think that it will also break the [transparent overlay logic](https://github.com/eclipse-theia/theia/blob/bfd9e0315dd8335c4e18005a2978fbe3b92053f0/packages/plugin-ext/src/main/browser/webview/webview.ts#L185-L194) because `mousedown` is not followed by `mouseup` event in this case.
Why not listening to a real `mousedown` and `mouseup` events instead? That is what I did in #8633.
Another concern: `mousedown` event is fired with no parameters which means `clientX` and `clientY` are 0,0. It means that in case some menu is opened at these coordinates it won't be closed.